Userland/Utilities/json.cpp

```diff
@@ -6,21 +6,49 @@
  */
 
 #include <AK/Assertions.h>
+#include <AK/Error.h>
 #include <AK/JsonArray.h>
 #include <AK/JsonObject.h>
 #include <AK/JsonValue.h>
 #include <AK/StringBuilder.h>
+#include <AK/StringUtils.h>
 #include <AK/StringView.h>
 #include <AK/Types.h>
 #include <AK/Vector.h>
 #include <LibCore/ArgsParser.h>
 #include <LibCore/File.h>
 #include <LibCore/System.h>
 #include <LibMain/Main.h>
+#include <limits.h>
 #include <unistd.h>
 
-static JsonValue query(JsonValue const& value, Vector<StringView>& key_parts, size_t key_index = 0);
+class SingleValueKey : public RefCounted<SingleValueKey> {
+private:
+    StringView m_key;
+
```

Comment:
Private members should go before public members

Reply:
Done (assuming you mean public before private)